catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -178,8 +179,13 @@ protected Object executeWithSubject() throws Exception {
         console.println("Starting metacard export...");
         Instant start = Instant.now();
         List<ExportItem> exportedItems = doMetacardExport(zipFile, filter);
-        console.println("Metacards exported in: " + getFormattedDuration(start));
+        if (exportedItems.isEmpty()) {
+            console.println("No metacards found to export, exiting.");
+            FileUtils.deleteQuietly(zipFile.getFile());
```

Comment:
❓ I believe this is the only place where you close the file (implicitly, with the `deleteQuietly`), so in the case where there are metacards exported, you'll leave it open. Might be best to use try-with-resources or wrap in a try/finally.